framework/sessions/__init__.py

```diff
@@ -77,6 +76,7 @@ def prepare_private_key():
 
 
 def get_session():
+    Session = apps.get_model('osf.Session')
```

Comment:
I agree with @cwisecarver . I'll take the blame for this, though--I think I started using apps.get_model during Django phase 1. Thinking it through now, though, using imports seems better so we don't break static analysis.

That said, I think this is not a blocker for merging this.